tests/installation/test_executor.py

```diff
@@ -252,11 +252,23 @@ def test_execute_works_with_ansi_output(
     env._run.assert_called_once()
 
     expected = [
-        "\x1b[39;1mPackage operations\x1b[39;22m: \x1b[34m1\x1b[39m install, \x1b[34m0\x1b[39m updates, \x1b[34m0\x1b[39m removals",
-        "\x1b[34;1m•\x1b[39;22m \x1b[39mInstalling \x1b[39m\x1b[36mpytest\x1b[39m\x1b[39m (\x1b[39m\x1b[39;1m3.5.2\x1b[39;22m\x1b[39m)\x1b[39m: \x1b[34mPending...\x1b[39m",
-        "\x1b[34;1m•\x1b[39;22m \x1b[39mInstalling \x1b[39m\x1b[36mpytest\x1b[39m\x1b[39m (\x1b[39m\x1b[39;1m3.5.2\x1b[39;22m\x1b[39m)\x1b[39m: \x1b[34mDownloading...\x1b[39m",
-        "\x1b[34;1m•\x1b[39;22m \x1b[39mInstalling \x1b[39m\x1b[36mpytest\x1b[39m\x1b[39m (\x1b[39m\x1b[39;1m3.5.2\x1b[39;22m\x1b[39m)\x1b[39m: \x1b[34mInstalling...\x1b[39m",
-        "\x1b[32;1m•\x1b[39;22m \x1b[39mInstalling \x1b[39m\x1b[36mpytest\x1b[39m\x1b[39m (\x1b[39m\x1b[32m3.5.2\x1b[39m\x1b[39m)\x1b[39m",  # finished
+        "\x1b[39;1mPackage operations\x1b[39;22m: \x1b[34m1\x1b[39m install,"
+        " \x1b[34m0\x1b[39m updates, \x1b[34m0\x1b[39m removals",
+        "\x1b[34;1m•\x1b[39;22m \x1b[39mInstalling"
+        " \x1b[39m\x1b[36mpytest\x1b[39m\x1b[39m"
+        " (\x1b[39m\x1b[39;1m3.5.2\x1b[39;22m\x1b[39m)\x1b[39m:"
+        " \x1b[34mPending...\x1b[39m",
+        "\x1b[34;1m•\x1b[39;22m \x1b[39mInstalling"
+        " \x1b[39m\x1b[36mpytest\x1b[39m\x1b[39m"
+        " (\x1b[39m\x1b[39;1m3.5.2\x1b[39;22m\x1b[39m)\x1b[39m:"
+        " \x1b[34mDownloading...\x1b[39m",
+        "\x1b[34;1m•\x1b[39;22m \x1b[39mInstalling"
+        " \x1b[39m\x1b[36mpytest\x1b[39m\x1b[39m"
+        " (\x1b[39m\x1b[39;1m3.5.2\x1b[39;22m\x1b[39m)\x1b[39m:"
+        " \x1b[34mInstalling...\x1b[39m",
+        "\x1b[32;1m•\x1b[39;22m \x1b[39mInstalling"
+        " \x1b[39m\x1b[36mpytest\x1b[39m\x1b[39m"
+        " (\x1b[39m\x1b[32m3.5.2\x1b[39m\x1b[39m)\x1b[39m",  # finished
```

Comment:
IMO the original formatting is better readable. Maybe we can agree as a rule of thumb to turn off formatting (like shown in https://stackoverflow.com/a/58584557/9750706) for expected strings in test in general?

Reply:
I agree that the result here is not very readable here. I can fix this one and the one in `tests/utils/test_helpers.py`; the autoformatting is fine in all other cases as far as I can see.

Most expected strings are multi-line strings, which `black` doesn't touch. Do you wish to keep the original formatting there as well? I can revert my manual changes. Ignoring errors for these is actually quite easy by putting a `# noqa: E501` after the final `"""`.

Reply:
LGTM :+1: